week-3/todo-list/script.js

```diff
@@ -1,8 +1,54 @@
+let list = document.getElementById("todo-list");
+
 function populateTodoList(todos) {
-  let list = document.getElementById("todo-list");
   // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+
+  for (const item of todos) {
+    addItem(item.task);
+  }
+}
+
+function addItem(task) {
+  if (!task){
+    alert("To-do is required!");
+    return;
+  }
+
+  const item = document.createElement("li");
+  item.innerText = `${task} `;
+
+  const btnSpan = document.createElement("span");
+  btnSpan.setAttribute("class", "badge bg-primary rounded-pill");
+
+  const iCheck = document.createElement("i");
+  iCheck.setAttribute("class", "fa fa-check");
+  iCheck.addEventListener("click", function () {
+    const classList = item.classList;
+    classList.contains("done")
+      ? classList.remove("done")
+      : classList.add("done");
```

Comment:
Could use .toggle() here if you wanted next time - it's a helpful function to save writing extra code https://stackoverflow.com/questions/18880890/how-do-i-toggle-an-elements-class-in-pure-javascript